ddl/db_integration_test.go

```diff
@@ -1257,56 +1257,6 @@ func (s *testIntegrationSuite3) TestResolveCharset(c *C) {
 	c.Assert(tbl.Meta().Charset, Equals, "binary")
 }
 
-func (s *testIntegrationSuite2) TestAddAnonymousIndex(c *C) {
```

Comment:
Why remove these tests?

Reply:
It's duplicated with `(s *testDBSuite3) TestAddAnonymousIndex`. This is written in the PR description.